SpywareFilter/sections/specific.txt

```diff
@@ -19,6 +19,9 @@
 ||tags.tiqcdn.com/utag/*/prod/utag.js$domain=radissonhotels.com|post.ch|usnews.com|yemeksepeti.com|krymr.com
 !
 !
+! https://github.com/AdguardTeam/Scriptlets/issues/163
```

Comment:
also add `TODO` comment with the link to the issue in Scriptlets so as not to forget about to remove `~domain`

Reply:
Ok, done.